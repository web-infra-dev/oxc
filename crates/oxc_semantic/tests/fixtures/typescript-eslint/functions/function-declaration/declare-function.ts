declare function A(): void;
function B(): void { };
