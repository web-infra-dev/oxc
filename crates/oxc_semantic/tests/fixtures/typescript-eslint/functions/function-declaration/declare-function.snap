---
source: crates/oxc_semantic/tests/main.rs
assertion_line: 147
input_file: crates/oxc_semantic/tests/fixtures/typescript-eslint/functions/function-declaration/declare-function.ts
snapshot_kind: text
---
[
  {
    "children": [
      {
        "children": [],
        "flags": "ScopeFlags(StrictMode | Function)",
        "id": 1,
        "node": "Function(A)",
        "symbols": []
      },
      {
        "children": [],
        "flags": "ScopeFlags(StrictMode | Function)",
        "id": 2,
        "node": "Function(B)",
        "symbols": []
      }
    ],
    "flags": "ScopeFlags(StrictMode | Top)",
    "id": 0,
    "node": "Program",
    "symbols": [
      {
        "flags": "SymbolFlags(BlockScopedVariable | Function)",
        "id": 0,
        "name": "A",
        "node": "Function(A)",
        "references": []
      },
      {
        "flags": "SymbolFlags(BlockScopedVariable | Function)",
        "id": 1,
        "name": "B",
        "node": "Function(B)",
        "references": []
      }
    ]
  }
]
